test/jdk/java/nio/Buffer/LimitDirectMemory.java

```diff
@@ -25,7 +25,7 @@
  * @test
  * @bug 4627316 6743526
  * @summary Test option to limit direct memory allocation
- * @requires (os.arch == "x86_64") | (os.arch == "amd64")
+ * @requires (os.arch == "x86_64") | (os.arch == "amd64") | (os.arch == "aarch64")
```

Comment:
Hello Brian, this seems to cover all the platforms that mainline currently supports. So I'm not sure if the `@requires` is anymore needed. I couldn't find the historical context on why that `@requires` was added in the first place.

Reply:
Never mind, I just realized that I hadn't taken into account other architectures like `riscv64`, `ppc64le` and similar. Although this test might still pass there, I think it's reasonable to continue to use the `@requires` like you have done here.

Reply:
Thanks, @jaikiran, for noticing this. I don't know the whole story about `@requires` here and this combination is tested.